examples/account/multiple_identities.rs

```diff
@@ -46,20 +46,37 @@ async fn main() -> Result<()> {
   // We can load the identity from storage into an account using the builder.
   let mut account1: Account = builder.load_identity(iota_did1).await?;
 
-  // Now we can modify the identity.
-  account1
-    .update_identity()
-    .create_method()
-    .fragment("my-key")
-    .apply()
-    .await?;
+  let account1_did: IotaDID = account1.did().clone();
+
+  // Now we can make modifications to the identity.
+  // We can even do so concurrently by spawning tasks.
+  let task1 = tokio::spawn(async move {
+    account1
+      .update_identity()
+      .create_method()
+      .fragment("my-key")
+      .apply()
+      .await
+  });
+
+  let task2 = tokio::spawn(async move {
+    account2
+      .update_identity()
+      .create_method()
+      .fragment("my-other-key")
+      .apply()
+      .await
+  });
+
+  task1.await.expect("task1 failed to execute to completion")?;
+  task2.await.expect("task2 failed to execute to completion")?;
```

Comment:
Apparently `join` is still suggested to await both tasks but it really doesn't matter since it doesn't affect the parallelism.

https://docs.rs/tokio/latest/tokio/macro.join.html#runtime-characteristics
> If parallelism is required, spawn each async expression using [tokio::spawn](https://docs.rs/tokio/latest/tokio/task/fn.spawn.html) and pass the join handle to join!.

No changes requested, just wanted to drop a link to the quote.